src/main/java/eu/enmeshed/model/messages/Message.java

```diff
@@ -27,5 +28,6 @@ public class Message extends ContentWrapper<MessageContent> implements WebhookDa
 
   private List<String> attachments;
 
-  private Boolean isOwn;
+  @JsonProperty("isOwn")
+  private boolean own;
```

Comment:
Why not `isOwn`?

Reply:
it will be isOwn in the getter and the naming would clash.